onboarding.md

```diff
@@ -61,52 +60,78 @@ This is a checklist for onboarding new members, workers, and contractors to the
 1. Add member SIN to Passbolt and share access with `People Operations` group.
    Change the member access to the entry as read-only, as this information is for recording keeping and the Member should not be able to modify it
 
-1. Collect encrypted (see [Sensitive Data](./guides.md#sensitive-data) guide) [`TD1` and `TD1-ON` forms](https://www.canada.ca/en/revenue-agency/services/forms-publications/td1-personal-tax-credits-returns.html) to [Drive](https://link.hypha.coop/drive)
+1. Collect encrypted (see [Sensitive Data](/guides/sensitive-data.md) guide) [`TD1` and `TD1-ON` forms](https://www.canada.ca/en/revenue-agency/services/forms-publications/td1-personal-tax-credits-returns.html) to [Drive](https://link.hypha.coop/drive)
 
-1. Add member to Clockify and [as Employee in Wagepoint](./guides.md#adding-a-new-employee) for payroll
+1. Add member to Clockify and [as Employee in Wagepoint](/guides/payroll.md#adding-a-new-employee) for payroll
 
 #### Set up communications
+##### Google Account
```

Comment:
The information here supersedes the info under the email  guide. We can change now or later but this is the up to date process.